src/autorenew-invite/autorenew-invite.constant.js

```diff
@@ -0,0 +1 @@
+export const SUBSIDIARIES_WITH_OPTIONAL_AUTORENEW = ['ASIA', 'AU', 'CA', 'FR', 'QC', 'SG', 'WE', 'WS'];
```

Comment:
Waiting for confirmation this is the correct list

Reply:
Should add line break.